### ISSUE TITLE
Conversion failed for "Object". Conversion failed for "model". Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by milewski** [Romakita/ts-express-decorators#382]
The latest version: `4.19.0` has also introduced a new bug on my project with the convertors... returning a model now is throwing the following error for me:

`Conversion failed for "Object". Conversion failed for "model". Cannot read property \'get\' of undefined` 

if i call `toObject()` before returning it works.. 

> will throw the mentioned error above
```ts
@Post('/plan/create')
@Status(201)
async create(@BodyParams() data: AdminPlanCreateRequest) {
    return {
        plan: await this.plansService.create(data)
    }
}
```

> works but wont use the convertors
```ts
@Post('/plan/create')
@Status(201)
async create(@BodyParams() data: AdminPlanCreateRequest) {
    return {
        plan: (await this.plansService.create(data)).toObject()
    }
}
```

**Comment by Romakita** [comment]
Yes, I'm on the fix ;)

**Comment by Romakita** [comment]
Will be fixed with the 4.20.0

**Comment by milewski** [comment]
thanks it works 👍
